src/frontend/src/constant/apiRoutes.ts

```diff
@@ -12,4 +12,7 @@ export const API_ROUTES = {
   GET_PIA_INTAKE: '/api/pia-intake/:id',
   PATCH_PIA_INTAKE: '/api/pia-intake/:id',
   GET_PIA_COMMENTS: '/api/comments',
```

Comment:
I think you can remove this route as the one you've created below is identical and it's name has greater semantic range. The `getComments` function will need to be updated to use the other one if this change is made.

I'll leave it up to you. I'm only mentioning it in an effort to reduce redundancy.

Reply:
done